README.md

```diff
@@ -1 +1,152 @@
-# Hello_World
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="https://github.com/microverseinc/readme-template/raw/master/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
[OPTIONAL] Remove the Microverse logo from the README file. You have to customize it for your project.

Reply:
Hello @RaoAkif 
I have modified it and proceed to make the merge.

Reply:
Perfect 👌🏻